Tests/XCUITests/IntegrationTests.swift

```diff
@@ -141,8 +146,9 @@ class IntegrationTests: BaseTestCase {
 
         // Sync again just to make sure to sync after new name is shown
         app.buttons["Settings"].tap()
+        sleep(3)
```

Comment:
Instead of sleep you can use a waitForExistence selecting one element in the settings view.